charts/scalardl-audit/values.yaml

```diff
@@ -333,3 +333,9 @@ auditor:
   authentication:
     # -- Specify the authentication method of ScalarDL. Available value is "digital-signature" or "hmac".
     method: "digital-signature"
+
+  # -- Pod annotations for the scalardl-auditor deployment
+  podAnnotations: {}
```

Comment:
The `podAnnotations` is already implemented on the ScalarDB Cluster chart side.

Honestly, I don't know why the `podAnnotations` is not implemented on the ScalarDL chart side...

However, I think it would be better to add this `podAnnotations` configuration from the following two perspectives:

- The consistency between ScalarDB and ScalarDL charts.
- Sometimes (especially for the cloud marketplace dedicated settings), we might need to set arbitrary annotations to the Pod.